.github/workflows/csfix.yml

```diff
@@ -1,6 +1,7 @@
 on:
-  issues:
+  pull_request:
```

Comment:
Vllt ist das hier oben das einzige Problem?

Ich dachte der eigentliche job hätte bereits funktioniert..

Reply:
bei mir hatte der beim Setzen des Labels hier nicht getriggert. Durch diese Änderung erst.

Das aktuelle Problem ist, dass der die Änderung nicht in diesem Branch comitten will.
Aktuell bin ich am überlegen, dass einfach plain in der Action selbst zu schreiben.

Und da man in Forks nur Read-only unterwegs ist, sollte das einfach sein, dass für dieses Repo speziell zu schreiben.

Reply:
Gerne probieren

Reply:
Das mit dem committen hat schonmal funktioniert ... hmm